doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -22,12 +22,34 @@ Required Packages
 -   [optional] CUDA Toolkit 6.5 or higher
 
 The packages can be installed using a terminal and the following commands or by using Synaptic
-Manager:
+Manager in Ubuntu 10.04, 16.04 LST:
 @code{.bash}
 [compiler] sudo apt-get install build-essential
 [required] sudo apt-get install cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev
 [optional] sudo apt-get install python-dev python-numpy libtbb2 libtbb-dev libjpeg-dev libpng-dev libtiff-dev libjasper-dev libdc1394-22-dev
 @endcode
+
+Use the following steps for openSUSE Leap 15.2 or by using Yast:
+@code{.bash}
+[compiler] sudo zypper in patterns-devel-C-C++-devel_C_C++
+[required] sudo zypper in gtk2-devel cmake pkgconfig ffmpeg-4-libavcodec-devel ffmpeg-4-libavformat-devel ffmpeg-4-libswscale-devel 
```

Comment:
Why FFMpeg? It's optional dependency.

Reply:
> Why FFMpeg? It's optional dependency.
Hi @dkurt ,

In openSUSE Leap 15.2, the package libavcodec-devel it must be in conjunction with ffmpeg. Bellow evidence:

# zypper se libavcodec-devel
Loading repository data...
Reading installed packages...

S | Name                      | Summary                                      | Type
--+---------------------------+----------------------------------------------+--------
  | ffmpeg-3-libavcodec-devel | Development files for FFmpeg's codec library | package
  | ffmpeg-4-libavcodec-devel | Development files for FFmpeg's codec library | package